receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -287,9 +298,9 @@ func consumeTraces(
 	return jbsr, numSpans, consumerError
 }
 
-func (jr *jReceiver) SubmitBatches(batches []*jaeger.Batch, options handler.SubmitBatchOptions) ([]*jaeger.BatchSubmitResponse, error) {
-	ctx := obsreport.ReceiverContext(
-		context.Background(), jr.instanceName, collectorHTTPTransport, collectorReceiverTagValue)
+func (jr *jReceiver) SubmitBatches(ctx context.Context, batches []*jaeger.Batch, options handler.SubmitBatchOptions) ([]*jaeger.BatchSubmitResponse, error) {
+	ctx = obsreport.ReceiverContext(
+		ctx, jr.instanceName, collectorHTTPTransport, collectorReceiverTagValue)
```

Comment:
This now also enables us to use the real HTTP request context in traces and in chained consumer.